subscription-engine/kafka-ws/app/Redis/RedisExpireMgmt.js

```diff
@@ -22,12 +26,7 @@ class RedisExpireMgmt {
     //   ...
     // };
 
-    const redisOptions = {
-      host: redisConfig.host,
-      port: redisConfig.port,
-      db: redisConfig.database,
-      // tls: tls_options
-    };
+    const redisOptions = { ...this.config };
```

Comment:
This is not necessary. You already have the object in the `this.config` variable.